test_helpers/src/c_util_test_env_params.c

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#include <stdlib.h>
+#include "c_util_test_helpers/c_util_test_env_params.h"
+
+#define DEFAULT_LEAK_LOG_FOLDER_NAME "g$\\blockstorage"

```

Comment:
>blockstorage [](http://example.com/codeflow?start=42&length=12)

ha, uh block store is "leaking" into c_utils (here and below)

Reply:
Is there really value in having a common environment variable like this across projects?

Reply:
You're right.  I was trying to limit code changes but it's cleaner to pass the root path in. I'm going to update this.